account_financial_risk/models/res_partner.py

```diff
@@ -135,7 +135,21 @@ class ResPartner(models.Model):
     )
     credit_policy = fields.Char()
     risk_allow_edit = fields.Boolean(compute="_compute_risk_allow_edit")
-    credit_limit = fields.Float(tracking=True)
+    credit_limit = fields.Float(string="Credit Limit (Original)")
```

Comment:
According what you said:

```suggestion
    credit_limit = fields.Float(string="Credit Limit (Original)", tracking=False)
```

Reply:
Oh, I see the point... forget it.